src/randoop/operation/Operation.java

```diff
@@ -0,0 +1,121 @@
+package randoop.operation;
+
+import java.io.PrintStream;
+import java.util.List;
+
+import randoop.ExecutionOutcome;
+import randoop.reflection.ReflectionPredicate;
+import randoop.sequence.Variable;
+
+/**
+ * Represents an operation (in the term algebra sense) that can occur in a statement 
```

Comment:
Citation for people who don't understand term algebra?  Or examples?  I'd like to keep the code readable for ordinary programmers.